test/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -387,6 +387,22 @@ public void getLocalDateWithError() {
         BridgeUtils.getLocalDateOrDefault("2017-05-10T05:05:10.000Z", null);
     }
     
+    @Test
+    public void toSynapseFriendlyName() {
+        assertEquals("This is a .-_ synapse Friendly Name3",
+                BridgeUtils.toSynapseFriendlyName("This (is a).-_ synapse Friendly Name3 "));
+    }
+    
+    @Test(expected = NullPointerException.class)
+    public void nullToSynapseFriendlyNameThrowsException() {
+        assertEquals("", BridgeUtils.toSynapseFriendlyName(null));
+    }
```

Comment:
Minor: No need for the assert here or in emptyStringToSynapseFriendlyName(), because BridgeUtils will throw before the assert is called.

Reply:
Yep, removed these